assets/src/infrastructure/ui/inputs/dateTime/Datepicker/index.tsx

```diff
@@ -1,35 +1,40 @@
 import React, { useCallback } from 'react';
 import classNames from 'classnames';
-import { Datepicker as BaseUIDatepicker } from 'baseui/datepicker';
+import { default as ReactDatepicker } from 'react-date-picker';
 
-import { DatepickerProps } from '../types';
-import withBaseProvider from '../withBaseProvider';
-import { useMemoStringify } from '@application/services/hooks';
+import { DatePickerProps } from '../types';
+import { CalendarOutlined, CloseOutlined } from '@appDisplay/icons/svgs';
 
 import './style.scss';
 
-const Datepicker: React.FC<DatepickerProps> = ({ value, onChange, onChangeValue, ...props }) => {
+const Datepicker: React.FC<DatePickerProps> = ({ value, onChange, onChangeValue, ...props }) => {
 	const className = classNames('ee-input-base-wrapper ee-date-picker', props.className);
-	const onChangeHandler: DatepickerProps['onChange'] = useCallback(
-		({ date }) => {
+	const onChangeHandler: DatePickerProps['onChange'] = useCallback(
+		(date) => {
 			if (typeof onChangeValue === 'function') {
 				onChangeValue(date as Date);
 			}
 
 			if (typeof onChange === 'function') {
-				onChange({ date });
+				onChange(date);
 			}
 		},
 		[onChange, onChangeValue]
 	);
 
-	const dateValue = useMemoStringify([value]);
-
 	return (
 		<div className={className}>
-			<BaseUIDatepicker {...props} onChange={onChangeHandler} value={dateValue} />
+			<ReactDatepicker
+				format={'y-MM-dd'}
```

Comment:
better to use a format from `constants` package.

Reply:
kk will do